modules/medicines/medicineService.js

```diff
@@ -0,0 +1,103 @@
+const HttpStatus = require('http-status-codes');
+const medicineQueryBulider = require('./medicineQueryBulider');
+const { validationResult } = require('express-validator');
+
+const getMedicine = async (req, res, next) => {
+  try {
+    const getMedicineResponse = await medicineQueryBulider.getMedicine(req);
```

Comment:
Namings should have a meaning. Here the queryBuilder returns list of medicines so 
const medicines = await medicineQueryBuilder.getMedicinesList(req)
is much more readable